css-canvas-medical/styles.css

```diff
@@ -0,0 +1,241 @@
+.container {
+  max-width: 80%;
+  margin: 10px auto;
+  font-family: "normal", Avenir, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+  display: flex;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+#header-link {
+  justify-content: space-evenly;
+}
+
+.header-img img {
+  width: 60%;
+  padding-left: 10%;
+}
+
+.header {
+  padding: 2rem 2rem 1rem;
+  border-bottom: 3px solid #b0b0b0;
+  display: flex;
```

Comment:
Why not just use the `row` class to accomplish this? You can add this class in addition to the `row` class to the element.